Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+
+        public static void Demo()
+        {
+            string name = GetString("What's your first name?");
+            string surname = GetString("What's your surname?");
+            int age = GetInt("How old are you?");
+            float weight = GetFloat("How much do you weight in kg?");
+            float height = GetFloat("How tall are you in cm?");
+            float mHeight = ConvertToMetres(height);
+            float bodyMassIndex = CalculateBMI(mHeight, weight);
+
+            Console.WriteLine(name + " " + surname + " is " + age + " years old, his weight is " + weight + "kg and his height is " + height + " cm");
+
+            Console.WriteLine("Their BMI is: " + bodyMassIndex);
+        }
+
+        public static string GetString(string message)
+        {
+            Console.WriteLine(message);
+            return Console.ReadLine();
+        }
+
+        public static int GetInt(string message)
```

Comment:
PromptInt would be a better name, because you ask to get an int, you don't just get it.

Reply:
Promp also suggests that you get it from some sort of a terminal (console)